packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -0,0 +1,302 @@
+import { IStyle, HighContrastSelector, ScreenWidthMaxSmall, getScreenSelector, getFocusStyle } from '../../Styling';
+import { IMessageBarStyleProps, IMessageBarStyles, MessageBarType } from './MessageBar.types';
+
+export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
+  const { theme, className, messageBarType, onDismiss, actions, truncated, isMultiline, expandSingleLine } = props;
+  const { semanticColors, palette, fonts } = theme;
+
+  const SmallScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
+
+  const errorStyle = {
+    background: semanticColors.errorBackground,
+    selectors: {
+      '& .icon': {
+        color: semanticColors.errorText
+      }
+    }
+  };
+
+  const blockedStyle = {
+    background: semanticColors.errorBackground,
+    selectors: {
+      '& .icon': {
```

Comment:
I'm not sure if this is the right way to do it. Which is not to say it is the wrong way, but another approach is to pass down color: semanticColors.errorText into the getStyles function of Icon. This is the problem Jason is grappling with in textfield and there's a similar situation with dropdown... This might be good enough for now.